src/MusicStore/samples/MusicStore/MusicStore.csproj

```
@@ -9,33 +9,35 @@
     <Configurations>Debug;Release;RuntimeStore</Configurations>
   </PropertyGroup>
 
+  <Import Project="$(RepositoryRoot)src\Servers\IIS\build\testsite.props" />
+
   <ItemGroup>
     <Content Update="ForTesting\**\*" CopyToPublishDirectory="Never" Condition=" '$(PublishForTesting)' != 'true' " />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp' AND '$(Configuration)' == 'RuntimeStore' ">
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="$(MicrosoftAspNetCoreAppPackageVersion)" />
+    <Reference Include="Microsoft.AspNetCore.All" />
```

Microsoft.AspNetCore.All was deprecated in 3.0.

```suggestion
    <Reference Include="Microsoft.AspNetCore.App" />
```